ansible/roles/stack-sunbird/templates/gql.env

```diff
@@ -0,0 +1,18 @@
+ENV=prod
+DATABASE_DEBUG=true
+HASURA_GRAPHQL_ADMIN_SECRET=HaTUNsNSaU
```

Comment:
No default passwords in public repos

Reply:
I have removed it for now, we have few more keys that need to be included in private repo and we will add those variables here.